extensions/blocks/simple-payments/index.js

```diff
@@ -12,6 +12,11 @@ import edit from './edit';
 import save from './save';
 import { DEFAULT_CURRENCY } from './constants';
 
+/**
+ * Example image
+ */
+import simplePaymentsExample1 from './simple-payments__example-1.jpg';
```

Comment:
Is there a specific reason why you used 2 underscores here? You used only one for the Tiled Galleries examples:
`extensions/blocks/tiled-gallery/tiled-gallery_example-1.jpg`

Maybe we should stick to that to remain consistent?

Reply:
Done in 3e7154719944a02dafa52162ceecc28b361d7408